jerry-debugger/jerry-client-ws.py

```diff
@@ -728,6 +759,24 @@ def parse_source(debugger, data):
     else:
         logging.debug("No pending breakpoints")
 
+def printSource(self, line_num):
+    last_bp = self.debugger.last_breakpoint_hit
+    lines = last_bp.function.source
+    if last_bp:
+        print("Source: %s" % (last_bp.function.source_name))
```

Comment:
please check whether `last_bp.function.source_name` is available. There is no point printing empty lines.